lib/reporting/identity_verification_report.rb

```diff
@@ -95,22 +98,41 @@ def as_csv
       csv << []
       csv << ['Metric', '# of Users']
       csv << []
-      csv << ['Started IdV Verification', idv_started]
-      csv << ['Submitted welcome page', idv_doc_auth_welcome_submitted]
-      csv << ['Images uploaded', idv_doc_auth_image_vendor_submitted]
+
+      collect_data(csv, data['all'])
+
+      if @ab_tests.present?
+        @ab_tests.keys.each do |ab_test|
+          @ab_tests[ab_test].each do |bucket|
+            csv << []
+            csv << ['A/B test', ab_test]
+            csv << ['bucket', bucket]
```

Comment:
I feel like this data might be more consumable if numbers for buckets were side-by-side, e.g. like this:



|  | Bucket A | Bucket B |
| -- | -- | -- |
| Started IdV Verification | 100 | 200 |
| Submitted welcome page | 98 | 33 |

Reply:
I don't think this would be a blocker for landing though)

Reply:
Absolutely, but for the spike I just got the data into the email.